tests/spec/core/include-config-spec.js

```diff
@@ -47,4 +47,24 @@ describe("Core — Include config as JSON", function() {
       expect(text).toEqual(expected);
     }).then(done);
   });
+
+  it("should remove irrelevant config for caniuse feature",async () => {
```

Comment:
As it's a self contained test with its own document, let's move this to the caniuse-spec instead.

Reply:
Nit: don't forget to run prettier on it! :)